app/assets/javascripts/lib/requests.js

```diff
@@ -17,7 +17,13 @@ export function request (url, method, data = {}, options = {scroll: true}) {
       let location = response.request.responseURL;
 
       if ((window.location.href == location) || (method == 'delete')){
-        window.location.reload(true); // save scroll position for annotations
+        // saving scroll position 
+        if (navigator.userAgent.match('Firefox') != null){
```

Comment:
Not sure why this works/is necessary but it does. Must be something going on deeper behind the scenes...